src/data-sources/http/HttpSettings.tsx

```diff
@@ -92,6 +92,11 @@ export const HttpSettings = ( { mode, uuid, config }: SettingsComponentProps< Ht
 
 				<HttpAuthSettingsInput auth={ state.auth } onChange={ handleAuthOnChange } />
 			</DataSourceForm.Setup>
+			<DataSourceForm.Blocks
+				handleOnChange={ handleOnChange }
+				// TO DO
+				hasEnabledBlocks={ true }
+			/>
```

Comment:
We can just remove this, right? No auto-generated config here.

Reply:
Yup, good catch! Was in auto-pilot here